osc/build.py

```diff
@@ -695,7 +702,7 @@ def main(apiurl, opts, argv):
         s = ''
         for i in opts.define:
             s += "%%define %s\n" % i
-        build_descr_data = s.encode() + build_descr_data
+        build_descr_data = s.encode + build_descr_data
```

Comment:
can you please add the () to the encode method call? Otherwise it breaks when using build --define ...